src/app/pages/send/send-crypto-asset-form/form/sip10/use-sip10-send-form.tsx

```diff
@@ -21,19 +20,19 @@ import { useSendFormNavigate } from '../../hooks/use-send-form-navigate';
 import { useStacksCommonSendForm } from '../stacks/use-stacks-common-send-form';
 
 interface UseSip10SendFormArgs {
-  symbol: string;
   contractId: string;
+  symbol: string;
 }
-export function useSip10SendForm({ symbol, contractId }: UseSip10SendFormArgs) {
-  const assetBalance = useStacksFungibleTokenAssetBalance(contractId);
-  const generateTx = useGenerateFtTokenTransferUnsignedTx(assetBalance);
+export function useSip10SendForm({ contractId, symbol }: UseSip10SendFormArgs) {
+  const asset = useSip010CryptoAsset(contractId);
```

Comment:
`sip10` or `sip010`?

Reply:
`useSip10SendForm` but `useSip010CryptoAsset`

Reply:
We should change all to `010` that is the standard SIP numbering:
https://github.com/stacksgov/sips

Reply:
@kyranjamie what are your thoughts here, I think using the right numbering makes sense, right? I was a bit held up when deciding to also use 009 in name for nft models. 🤔

Reply:
Don't have a strong opinion, but consistency is best. 

Think I have a slight pref for non-leading 0s as this is more natural language but up to you.